### ISSUE TITLE
[BUG - Release] Blueprint Mods not loading

### ISSUE THREAD

**Issue by MrGentle** [UE4SS-RE/RE-UE4SS#496]
**Branch or Release**
v3.0.1

**Game and Engine Version**
Dark And Darker, v5.3

**Describe the bug**
Blueprint Mods not loading with error message "Actor for mod '%s' is not valid\n".
Also, when putting the .pak files in a sub folder they're not even considered for loading.

**Mods directory**
I have 1 custom lua mod with a print statement. Shouldn't impact this.
```
local function log(text)
    print("[GentleMod] " .. tostring(text));
end

log("Initialized\n")
```

mods.txt
```
CheatManagerEnablerMod : 0
ActorDumperMod : 0
ConsoleCommandsMod : 1
ConsoleEnablerMod : 1
SplitScreenMod : 0
LineTraceMod : 0
BPModLoaderMod : 1
BPML_GenericFunctions : 1
jsbLuaProfilerMod : 0
GentleMod: 1



; Built-in keybinds, do not move up!
Keybinds : 1
```

**To Reproduce**
Follow any of the available BlueprintMod guides (Russells guide, PalWorld guide, Hogwarts Legacy guide )

**Expected behavior**
Expected mod to load

**Screenshots, UE4SS Log, and .dmp file**
```
[17:31:03] Starting Lua mod 'BPModLoaderMod'
[17:31:03] [Lua] Mods/BPModLoaderMod/load_order.txt not present or no matching mods, loading all BP mods in random order.
[17:31:03] [Lua] UITest == table: 000001A28F110420
[17:31:03] [Lua]     AssetName == ModActor_C
[17:31:03] [Lua]     AssetNameAsFName == FNameUserdata: 000001A2F903DFF8
[17:31:03] [Lua]     Name == UITest
[17:31:03] [Lua]     AssetPath == /Game/Mods/UITest/ModActor
[17:31:03] Starting Lua mod 'BPML_GenericFunctions'
[17:31:03] Mod 'jsbLuaProfilerMod' disabled in mods.txt.
[17:31:03] Starting Lua mod 'Keybinds'
[17:31:03] Starting mods (from enabled.txt, no defined load order)...
[17:31:03] Event loop start
[17:31:08] [Lua] [ConsoleEnabler] ConsoleKey[1]: F10
[17:31:09] [Lua] Loading mod: UITest
[17:31:09] [Lua] Actor for mod 'UITest' is not valid
```
If "use io" is checked
```
[17:38:53] [Lua] Loading mod: UITest
[17:38:53] [Lua] ModClass for 'UITest' is not valid
ObjectPath: 
PackageName: /Game/Mods/UITest/ModActor
AssetName: ModActor_C
```
**Desktop (please complete the following information):**
Win 11

**Additional context**
![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/20410119/4ece4300-3db2-482f-b74a-3c8e040e141a)
![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/20410119/e77802d9-6c21-4d3c-967c-4c208a18dda7)
They're all in the same chunk.

![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/20410119/7624a521-5640-4ff4-be7a-62c39ec166bd)

- Project name is the same as the games project name
- Project is set to export shipping
- Allow ChunkID assignments is on
- Generate chunks is on.
- Files are all set to the same chunk
- I have tried with and without "use io"
- I have tried with and without the CustomContent folder

**Comment by LastPlayerTR** [comment]
Same issue
Branch or Release
v3.0.1

Game and Engine Version
Ranch Simulator, UE5.2

**Comment by UE4SS** [comment]
This is likely to be fixed by #503.
I'll let you know when an experimental build is available for you to test.

**Comment by UE4SS** [comment]
I completely forgot about this, sorry.
Here's the experimental build: https://github.com/UE4SS-RE/RE-UE4SS/releases/download/experimental/zDEV-UE4SS_v3.0.1-81-gd8189f3.zip
